docs/design/core/metrics/MetricsList.md

```diff
@@ -1,130 +1,61 @@
-   
-2) What errors are considered as throttling errors?
-   The request was considered as throttled if one of the following conditions are met:
-   1) The http status code is equal to: `429` or `503`
-   2) The error code is equal to one of the following values:  
-   SlowDown   
-   SlowDownException  
-   Throttling  
-   ThrottlingException  
-   Throttled  
-   ThrottledException  
-   ServiceUnavailable  
-   ServiceUnavailableException  
-   ServiceUnavailableError  
-   ProvisionedThroughputExceededException  
-   TooManyRequests  
-   TooManyRequestsException  
-   DescribeAttachmentLimitExceeded
-
-           
-## References
-1) [V1 Metrics Description](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/metrics/package-summary.html)
+# Collected Metrics
+
+ This document lists the metrics collected by various components of the SDK,
+ namely the core libraries, and HTTP clients.
+
+ A note on collector path: The path is assuming a `MetricCollection` tree rooted at an API call.
+
+## Core Metrics
+
+The set of core metrics includes all metrics collected by the core components
+of the SDK. This includes components like SDK service clients,
+request/resposne marshallers and unmarshallers, and signers.
+
+All the in code constants associated with metric below can be found in the
+[`software.amazon.awssdk.core.metrics.CoreMetric`](https://github.com/aws/aws-sdk-java-v2/blob/8c192e3b04892987bf0872f76ba4f65167f3a872/core/sdk-core/src/main/java/software/amazon/awssdk/core/metrics/CoreMetric.java#L24)
+class within `sdk-core`.
+
+| Name                          | Type          | Description |
+|-------------------------------|---------------|-------------|
+| ServiceId                     | `String`      | The unique ID for the service. This is present for all API call metrics.|
+| OperationName                 | `String`      | The name of the service operation being invoked. This is present for all API call metrics.|
+| ApiCallDuration               | `Duration`    | The duration of the API call. This includes all call attempts made.|
+| ApiCallSuccessful             | `Boolean`     | True if the API call succeeded, false otherwise. |
+| BackoffDelayDuration          | `Duration`    | The duration of time that the SDK has waited before this API call attempt, based on the retry policy. |
+| MarshallingDuration           | `Duration`    | The duration of time taken to marshall the SDK request to an HTTP request.|
+| CredentialsFetchDuration      | `Duration`    | The duration of time taken to fetch signing credentials for the request.|
+| SigningDuration               | `Duration`    | The duration fo time taken to sign the HTTP request.|
+| HttpRequestRoundTripTime      | `Duration`    | The total time take to send a HTTP request and receive the response.|
+| AwsRequestId                  | `String`      | The extended request ID of the service request.|
```

Comment:
The description is the same as the next line, are they the same thing?

Reply:
Nice catch, fixed